packages/@react-spectrum/combobox/src/MobileComboBox.tsx

```diff
@@ -371,16 +415,19 @@ function ComboBoxTray(props: ComboBoxTrayProps) {
           inputProps={inputProps}
           inputRef={inputRef}
           isDisabled={isDisabled}
+          isLoading={loadingState === 'loading' || loadingState === 'filtering'}
+          loadingIndicator={loadingState != null && loadingCircle}
           validationState={validationState}
-          wrapperChildren={(state.inputValue !== '' && !props.isReadOnly) && clearButton}
+          wrapperChildren={!props.isReadOnly && clearButton}
           UNSAFE_className={
             classNames(
               searchStyles,
               'spectrum-Search',
               'spectrum-Textfield',
+              'spectrum-Search--loadable',
```

Comment:
Made it always have the loadable styles so that the validation icon and clear button for non-async comboboxes are ordered the same way for consistency (e.g. validation icon is now on the left and clear button is on the right). Note that this change is only for mobile combobox's search input and normal searchfields still have the validation icon positioned to the right of the clear button.

Reply:
Open to naming it something different, couldn't think up of a better name